assets/scripts/util.js

```diff
@@ -332,14 +336,14 @@ function radio_trend(category, measured, target) {
   return CATEGORIES[category][2];
 }
 
-var DIRECTIONS = ["N", "NE", "E", "SE", "S", "SW", "W", "NW"];
+function radio_altitude(alt) {
+  if(alt >= 18000) return "flight level " + parseInt(alt.toString().substr(0,3));
```

Comment:
Even when fl 1000 might not be realistic I wouldn't advise to just take the first 3 digits (I prefer division to get that number). Also what about altitudes not rounded by 100?